### PR TITLE
Do not serial for tests of Pod Host IPs.

### PR DESCRIPTION
**Pull request by liangyuanpeng**
#### What type of PR is this?

/kind cleanup
#### What this PR does / why we need it:

show the tests of `Pod Host IPs` at testgrid for a while before actually deleting the feature gate of `PodHostIPs` to ensure that there won't be any problems.

https://github.com/kubernetes/kubernetes/pull/126752/files#r1722214824
/assign @SergeyKanzhelev

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: test/e2e_node/pod_host_ips.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ import (
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-var _ = common.SIGDescribe("Pod Host IPs", framework.WithSerial(), func() {
```
Comment (SergeyKanzhelev) [inline test/e2e_node/pod_host_ips.go:46]: Why was it serial in the first place? Are they somehow disruptive to other tests? Or may be interrupted by other tests?

Comment (pacoxu) [reply test/e2e_node/pod_host_ips.go:46]: When the FG is alpha, we add a kubelet configure step before running the test. At that time, it have to be serial.

But later, we can remove it when it is GA.

/lgtm
/assign @wzshiming

Comment (pacoxu) [reply test/e2e_node/pod_host_ips.go:46]: See https://github.com/kubernetes/kubernetes/pull/121429/ for more details when adding the e2e.

**Comment by pacoxu** [comment]
/priority important-soon
/triage accepted 
The FG was GAed in v1.30 and will be remove in this release cycle.

File: test/e2e_node/pod_host_ips.go (hunk 1 above)
Comment (liangyuanpeng) [reply test/e2e_node/pod_host_ips.go:46]: @pacoxu  I should have studied the history here before I removed the code, thank you for bringing the historical information here.

I have check the code and have not see any code need to serial now.

**Comment by aojea** [comment]
/approve

it seems Sergey question was addressed and now is safe to make it non serial 

https://github.com/kubernetes/kubernetes/pull/126810#discussion_r1743180790

File: test/e2e_node/pod_host_ips.go (hunk 1 above)
Comment (SergeyKanzhelev) [inline test/e2e_node/pod_host_ips.go:46]: Please mark it as NodeConformance. Otherwise it may not run in any of test lanes

**Pull request merged**